src/website/src/app/documentation/demos/tree-view/tree-view.demo.html

```diff
@@ -300,6 +300,21 @@ <h3 id="basic-tree">Basic Tree</h3>
                 To pre-expand a node, you can use the <code class="clr-code">[clrExpanded]</code> input.
             </p>
 
+            <div class="alert alert-info has-margin-top-b2">
+                <div class="alert-items">
+                    <div class="alert-item static">
+                        <div class="alert-icon-wrapper">
+                            <clr-icon class="alert-icon" shape="info-circle"></clr-icon>
+                        </div>
+                        <div class="alert-text">
+                            Please note that every tree require to have root node to work properly,
+                            this is done by having <code class="clr-code">clr-tree</code> as root wrapper, you
+                            could check the example below.
```

Comment:
nit: require -> requires
I'd also change "you could check" -> "please check", or even drop the entire last sub-sentence as it hijacks the goal of the example.